static/styles/night_mode.scss

```diff
@@ -978,7 +978,7 @@ on a dark background, and don't change the dark labels dark either. */
         }
     }
 
-    div.popover-content a:focus {
+    a:focus {
```

Comment:
This should have the not(:active) too, right?  

Probably worth a comment if we're not going to match the selector.

Reply:
it could but it doesn't matter since `outline-color` is ignored if there's no outline

Reply:
fixed